test/jit/test_tracer.py

```diff
@@ -900,6 +901,7 @@ def foo(a):
         self.assertEqual(foo(x), x + x + x)
 
     @unittest.skipIf(not RUN_CUDA, "calls .cuda()")
+    @with_tf32_off
```

Comment:
Add a comment here explaining why tf32 is disabled

Reply:
fixed